packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,64 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the pod_target_xcconfig definition in expo-dev-launcher.podspec
+
+- (NSURL * _Nullable)devLauncherURL
+{
+#ifdef EX_DEV_LAUNCHER_URL
+  NSString *urlString = [NSString stringWithFormat:@"%s/%s", EX_DEV_LAUNCHER_URL, EX_DEV_LAUNCHER_PACKAGER_PATH];
+  return [NSURL URLWithString:urlString];
```

Comment:
Tiny technical nit: the path part of a URL includes its leading slash character and perhaps the slash should be in EX_DEV_LAUNCHER_PACKAGER_PATH instead of the format string.

```objc
  NSString *urlString = [NSString stringWithFormat:@"%s%s", EX_DEV_LAUNCHER_URL, EX_DEV_LAUNCHER_PACKAGER_PATH];
```

But that said, NSURL has a web-like API for resolving URLs. I think it follows RFC 3986 Section 5 (though the Apple docs don't mention that RFC specifically and there could be some subtle differences but none that likely matter for our need here). It's the standard URL resolution behavior where `/a/b` is relative to the domain while `a/b/` preserves directories in the base URL.

```suggestion
  NSURL *baseUrl = [NSURL URLWithString:EX_DEV_LAUNCHER_URL];
  return [NSURL URLWithString:EX_DEV_LAUNCHER_PACKAGER_PATH relativeToURL:baseUrl];
```